src/HypLSP8Collateral.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.19;
+
+import { TokenRouter } from "@hyperlane-xyz/core/contracts/token/libs/TokenRouter.sol";
+
+import { TokenMessage } from "@hyperlane-xyz/core/contracts/token/libs/TokenMessage.sol";
+
+import { ILSP8IdentifiableDigitalAsset } from "@lukso/lsp8-contracts/contracts/ILSP8IdentifiableDigitalAsset.sol";
```

Comment:
Small suggestion but not mandatory. Maybe aliasing this import will make the code shorter and a bit more readable?

```suggestion
import { ILSP8IdentifiableDigitalAsset as ILSP8 } from "@lukso/lsp8-contracts/contracts/ILSP8IdentifiableDigitalAsset.sol";
```

So below you can write it like this (it's a bit shorter). But again, not a biggie:

```solidity

    function ownerOf(uint256 _tokenId) external view returns (address) {
        return ILSP8(wrappedToken).tokenOwnerOf(bytes32(_tokenId));
    }
```